css/styles.css

```diff
@@ -98,18 +98,18 @@ body {
 .btn {
   text-decoration: none;
   display: inline-block;
-  border: 2px solid #c761a2; 
```

Comment:
border should be 1px and the color should also be #ff52bf. I believe this color resembles the original style.